app/abilities/base_ability.rb

```diff
@@ -0,0 +1,6 @@
+class BaseAbility
+  include CanCan::Ability
+
+  def initialize(user)
+  end
```

Comment:
honestly.

It barely matters.

It's about readability.

If this were anyone else's code I wouldn't care because the method is empty so it doesn't need any space to help it be read - it simply needs to exist at this point.

If it had stuff in it an extra line of WS would help readability.

If there were two empty methods, I'd want one line of WS between the methods, wouldn't care about between method and end of class because it wouldn't effect readability